test/unit/uk/gov/hmrc/exports/services/mapping/governmentagencygoodsitem/AdditionalDocumentsBuilderSpec.scala

```diff
@@ -50,14 +50,14 @@ class AdditionalDocumentsBuilderSpec extends UnitSpec with GovernmentAgencyGoods
       mappedDocuments.get(0).getSubmitter.getName.getValue mustBe issusingAuthorityName
 
       val writeoff = mappedDocuments.get(0).getWriteOff
-      writeoff.getAmountAmount mustBe null
+      Option(writeoff.getAmountAmount) mustBe None
```

Comment:
Not sure if it makes a big difference and I agree that `null` should be avoided, but it has been used here due to the wco model being based on Java classes.
I am simplt not sure if this change may cause something to slip through?

Reply:
Not sure to understand the comment. If `writeoff.getAmountAmount` returns `null`, `Option` will take care of that and the value will be `None`. So no slip through. Isn't it?